examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldClientTls.java

```diff
@@ -110,6 +110,7 @@ public static void main(String[] args) throws Exception {
         HelloWorldClientTls client;
```

Comment:
```
[trustCertCollectionFilePath " +
                 "[clientCertChainFilePath clientPrivateKeyFilePath]]\n  Note: 
```

Otherwise, you may support `args.length == 4` or other alternative approaches.